frontend/packages/ux-editor/src/components/FormComponent/FormComponent.test.tsx

```diff
@@ -183,9 +199,7 @@ describe('FormComponent', () => {
         },
       });
 
-      expect(
-        screen.getByRole('img', { name: textMock('ux_editor.component_input') })
-      ).toBeInTheDocument();
+      expect(screen.getByTitle(textMock('ux_editor.component_input'))).toBeInTheDocument();
```

Comment:
I removed the role "img" from the SVG picture since it's just for looks and not for people who use screen-reading software. I also added "aria-hidden" to ensure screen readers don't read the icons. Then, I changed the way we query in the tests. Instead of getByRole," we now use "getByTitle" to find the "<title>InconTitle</title>" inside the SVG code.

  describe('icon', () => {
    it('should display the icon', async () => {
      await render({
        component: {
          ...component1Mock, // The mock is an input-field
          icon: 'Icon',
        },
      });

      expect(screen.getByTitle(textMock('ux_editor.component_input'))).toBeInTheDocument();
    });
  });
```

Might I misunderstood your question? 😊